roles/ceph-osd/tasks/activate_osds.yml

```diff
@@ -41,6 +41,18 @@
 - include: osd_fragment.yml
   when: crush_location
 
+- name: check if selinux is enabled
+  command: getenforce
+  register: sestatus
+  changed_when: false
```

Comment:
add `failed_when: false` and/or a `when: ansible_os_family == 'RedHat'` so if the command does not exist the play doesn't fail.